references/settings.md

```diff
@@ -78,17 +78,19 @@ An objet containing document attributes as keys and  `asc` ascending or `dsc` de
 
 #### Ranking order
 
-A list of ranking rules ordered by importance for the [bucket sort](/advanced_guides/bucket_sort). The first rule being the most important.
+A list of ranking rules ordered by importance for the [bucket sort](/advanced_guides/bucket_sort.md). The first rule being the most important.
 
-#### Distinct
+#### Distinct field
 
-A string containing the attribute that needs to be [distinct](/advanced_guides/distinct).
+A string containing the attribute that needs to be [distinct](/advanced_guides/distinct.md).
 
-::: warning
-None of the 3 settings parameters is mandatory
+::: note
+None of the 3 settings parameters are mandatory
 :::
 
-### Example
+### Examples
+
+#### Add settings
 
 ```bash
 curl \
```

Comment:
Why is there `"distinctField": "",` ? That's probably invalid.

Reply:
I don't know, it was there before.
I don't know what to write, to be honest, I have never used this setting before, and the current example does not help me ^^
Can you give me an example to replace that? :)

Reply:
(if by invalid you think it will not be processed, it is)

Reply:
By invalid I mean that it the `"distinctField"` paramter must not be set to empty but even `null` or not be specified at all.

Reply:
Oula, I don't understand at all.
must not be set to empty but even null  -> by default the value of `distinctField` is `null`
or not be specified at all -> why can we change it in the body of the request then?

Reply:
In this cURL example the `"DistinctField"` is set to empty (`""`), as yo wrote it in the paragraph above, setting it to empty is invalid, the engine will try to distinct documents on an empty, non-existent field which is invalid.

The API accept two things:
  - Set the `"DistinctField"` to `null` to reset its value to the default one
  - Set the `"DistinctField"` to a valid field, this way the engine will distinct documents on the specified existing field.
  - Don't write it in the body at all, this way the API will ignore and keep the current value associated to it.

Reply:
Yes, I well understood that the current example is not a great and consistent example, and that's why I asked you first to give me a valid field (your second point), so that I can put it as example documentation :)

Reply:
Removing the `"distinctField"` from the body should do the job.